docs/sources/enterprise/access-control/permissions.md

```diff
@@ -23,50 +23,50 @@ scope
+| `org.users.read`           | `users:*`                                                                               | Get user profiles within an organization.                                                                                                                |
+| `org.users.add`            | `users:*`                                                                               | Add a user to an organization.                                                                                                                           |
+| `org.users.remove`         | `users:*`                                                                               | Remove a user from an organization.                                                                                                                      |
+| `org.users.role:update`    | `users:*`                                                                               | Update the organization role (`Viewer`, `Editor`, `Admin`) for an organization.                                                                          |
+| `ldap.user:read`           | n/a                                                                                     | Get a user via LDAP.                                                                                                                                     |
+| `ldap.user:sync`           | n/a                                                                                     | Sync a user via LDAP.                                                                                                                                    |
+| `ldap.status:read`         | n/a                                                                                     | Verify the LDAP servers’ availability.                                                                                                                   |
+| `ldap.config:reload`       | n/a                                                                                     | Reload the LDAP configuration.                                                                                                                           |
+| `status:accesscontrol`     | `services:accesscontrol`                                                                | Get access-control enabled status.                                                                                                                       |
+| `settings:read`            | `settings:*`<br>`settings:auth.saml:*`<br>`settings:auth.saml:enabled` (property level) | Read settings                                                                                                                                            |
```

Comment:
Please fill in the blank: "Read the _____ ?______ settings."

Reply:
Not sure I understand? This line has changed because of automatic formatting

Reply:
```suggestion
| `settings:read`            | `settings:*`<br>`settings:auth.saml:*`<br>`settings:auth.saml:enabled` (property level) | Read the [Grafana configuration settings]({{< relref "../../administration/configuration/_index.md" >}})                                                                                                                                       |
```